packages/web-components/src/tooltip/tooltip.ts

```diff
@@ -0,0 +1,116 @@
+import { attr, FASTElement } from '@microsoft/fast-element';
+
+/**
+ * A Tooltip Custom HTML Element.
+ * @public
+ */
+export class Tooltip extends FASTElement {
+  public elementInternals = this.attachInternals();
+
+  /**
+   * Set the delay for the tooltip
+   */
+  @attr
+  public delay: number = 250;
+
+  /**
+   * Set the positioning of the tooltip
+   */
+  @attr
+  public positioning: string = 'above';
```

Comment:
Can this be treated as a default via CSS? IE, it's the default value for the host and every other instance is applied via specificity?

Reply:
Yeah. I can work on that. This originally made the cut because I needed to apply `--blockOffset` but I could do that with a `:not([positioning])` as well.